### ISSUE TITLE
doc_markdown false positive (as underscores are within a url)

### ISSUE THREAD

**Issue by theotherphil** [Manishearth/rust-clippy#1558]
```
/// Equalises the histogram of an 8bpp grayscale image in place.
/// https://en.wikipedia.org/wiki/Histogram_equalization
pub fn equalize_histogram_mut<I>(image: &mut I)
```

clippy wants me to put Histogram_equalization in backticks.

**Comment by mcarton** [comment]
Make a proper link and Clippy will be happy:

```rust
/// Equalises the histogram of an 8bpp grayscale image in place.
/// See also [Histogram equalization] on wikipedia.
/// [Histogram equalization]: https://en.wikipedia.org/wiki/Histogram_equalization
pub fn equalize_histogram_mut<I>(image: &mut I)
```
